Dockerfile.activator

```diff
@@ -9,6 +9,7 @@ ARG PORT
 ARG CALIB_REPO
 ARG IMAGE_BUCKET
 ARG IMAGE_TIMEOUT
+ARG BUCKET_TOPIC
```

Comment:
Not about this PR, but this makes me wonder why some env vars in activator aren't here and it works just fine...

Reply:
We probably should remove them then. I can create a ticket for that.

Reply:
[DM-39030](https://jira.lsstcorp.org/browse/DM-39030)